lib/jwk/rsakey.js

```diff
@@ -95,6 +96,21 @@ var JWKRsaCfg = {
   },
   verifyKey: function(alg, keys) {
     return keys.public;
+  },
+
+  convertToPEM: function(key, isPrivate) {
+    var k = rsau.convertToForge(key, !isPrivate);
+    if (!isPrivate) {
+      return forge.pki.publicKeyToPem(k);
+    }
+    return forge.pki.privateKeyToPem(k);
+  },
+  convertToDER: function(key, isPrivate) {
```

Comment:
I'm not sure about including the DER format right now.  There are a couple of different formats here, and I'm more inclined to use what is expected by WebCrypto (which is "SPKI" and "PKCS#8").   I believe forge emits the "raw" values from RFC 3447.